eclass/texlive-common.eclass

```diff
@@ -140,7 +140,7 @@ dobin_texmf_scripts() {
 
 etexmf-update() {
 	if has_version 'app-text/texlive-core' ; then
-		if [ -z "${ROOT%/}" ] && [ -x "${EPREFIX}"/usr/sbin/texmf-update ] ; then
+		if [[ -z ${ROOT%/} ]] && [[ -x "${EPREFIX}"/usr/sbin/texmf-update ]] ; then
```

Comment:
`%/` shouldn't be needed anymore on EAPI 7-only, and spurious double quotes in the second expression